### ISSUE TITLE
Does not work with docker-compose

### ISSUE THREAD

**Issue by lserman** [rfkrocktk/docker-puppetmaster#25]
This issue was fixed in Phusion's baseimage: https://github.com/phusion/baseimage-docker/issues/230 but still exists in the base image used for this repo's Dockerfile. 

It causes this error when running with compose:

```
puppetmaster_1 | /etc/container_environment.sh: line 7: export: `affinity:container==79aa799631f1735a1b3ae01220d60d69c2c6167c63b35f7
562d6ca1668a61747': not a valid identifier
```

**Comment by rfkrocktk** [comment]
I will be updating the image today.

Thanks,
 - Naftuli Tzvi

**Comment by naftulikay** [comment]
This has finally been finished, sorry for the delay. The container is built now on the latest Phusion baseimage. Fixed with v3.8.6-3.
